components/search/SearchSelectedFacets.vue

```diff
@@ -24,8 +24,16 @@
       facetList: function() {
         let listOfFacets = [];
         for (let facetName in this.facets) {
+
```

Comment:
In this component, the filter badge text should use the _label_ of the theme, not its query value, e.g. "1914-1918" not "ww1"